docs/src/transforms.md

```diff
@@ -122,6 +122,18 @@ While `system.peers`/`system.peers_v2` will be rewritten to list the configured
   # read_timeout: 60
 ```
 
+#### Error handling
+
+If shotover sends a request to a node and never gets a response, (maybe the node went down), shotover will return a cassandra `Server` error to the client.
+This is because the message may or may not have succeded, so only the client can attempt to retry as the retry may involve checking if the original query did in fact complete succesfully.
+
+If no nodes are capable of receiving the query then shotover will return a cassandra `Overloaded` error indicating that the client should retry the query at some point.
```

Comment:
*Shotover :P

Reply:
Cassandra should be capitalised too, I think?